pkg/verifier/package_verifier.go

```diff
@@ -39,6 +39,7 @@ type AwsEgressConfig struct {
 	KmsKeyID         string
 	SecurityGroupId  string // Deprecated: prefer securityGroupIDs
 	SecurityGroupIDs []string
+	VpcID            string
```

Comment:
Requiring this parameter is a non-trivial change to our API. Could we avoid having to ask the caller for this, e.g., by checking the VPC associated with the provided security group ID? Or we could note the VPC attached to the probe instance, perhaps?

Reply:
This is what I am doing @abyrne55  this param I am not requesting from the user. I am populating it with a function GetVpcIdFromSubnetId() and filling it. however, we can also change it for a variable in the function scope.

Reply:
Changed it to a regular variable after some thought.